README.md

```diff
@@ -11,3 +11,10 @@ flake8-f-string is a flake8 plugin to warn passing callable objects as arguments
 
 - Python 3
 - flake8 >= 3.5
+
+Error codes
+-----------
+
+This plugin uses the following error codes:
+
+- FS100 - passing callable objects as arguments for formatted string
```

Comment:
All the error codes start with `101`, not with `100`.

Let's not introduce new rules - keep existing.

Check this (or any plugin):
https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes